Week01/Problem01/klai_01.py

```diff
@@ -0,0 +1,2 @@
+l = [x for x in range(1,1000) if x%3 == 0 or x%5 == 0]
```

Comment:
List comprehensions are cool and concise, but they can be difficult to read for anyone looking back at your code.  Consider either (a) writing comments describing what you're doing, or (b) spreading this comprehension into multiple lines, as in:
```
list = []
for x in range(1,1000):
    if (x % 3 == 0) or (x % 5 == 0): 
        list.append(x)

print(sum(list))
```
This will make the lives of anyone maintaining your code (including yourself, six months from now!) much easier.  For now, where everything is pretty short and self-explanatory, it's not such a big deal, but as the workhsop progresses, our code may become sufficiently complicated that *both* comments and clearly organized code are important.